operator/webhook/MakefileWin

```diff
@@ -0,0 +1,48 @@
+VERSION ?= 0.7.0
+HOST_PORT ?= 7080
+GIT_TAG := webhook_v$(VERSION)
+
+IMG_REGISTRY := ghcr.io/octoconsulting
+IMG_NAME := keip/route-webhook
+FULL_IMAGE_NAME := $(IMG_REGISTRY)/$(IMG_NAME):$(VERSION)
+CONTAINER_NAME := integration-route-webhook
+PYTHON := python
+TEST_COVERAGE_DIR := .test_coverage
+TEST_COVERAGE_FILE := $(TEST_COVERAGE_DIR)/.coverage

```

Comment:
If we are going to commit to supporting a Makefile compatible with Windows, we should extract these shared variables to a common Makefile that can be included. 

Alternatively, it seems that the difference between the Unix and Windows Makefile is fairly small (for now), so we might be able to stick to a single Makefile with some conditionals that check for the OS and decide which version of a command to run, that can get ugly quickly though, but might be ok for something small like this.

Another potential option, is to just add a separate target for windows-specific tasks in the original Makefile (e.g. `test` and `wintest`)

Reply:
I added the Windows specific tasks to the Makefile and prefixed them with `win-`.